src/framework/audio/thirdparty/opus/opus-1.4/CMakeLists.txt

```diff
@@ -644,3 +644,7 @@ if(BUILD_TESTING AND NOT BUILD_SHARED_LIBS)
         -DCMAKE_SYSTEM_NAME=${CMAKE_SYSTEM_NAME}
         -P "${PROJECT_SOURCE_DIR}/cmake/RunTest.cmake")
 endif()
+
+#target_no_warning(${MODULE} -Wno-conversion)
+#target_no_warning(${MODULE} -Wno-truncate)
+#target_no_warning(${MODULE} -Wno-uninitialized)
```

Comment:
This CMakeLists file is part of Opus, so doesn't work with our "modularity" structure. So `${MODULE}` is either not defined at all here, or perhaps it's defined to the name of the "nearest ancestor" folder that does use our modularity. 

Also, because it is 3rdparty, we should not edit this CMakeLists file.

Solution: in `SetupOpus.cmake`, just after `set(OPUS_LIBRARIES opus)`, add:
```cmake
target_no_warning(opus -Wno-conversion)
…
```
Note that `-Wno-truncate` is not yet mentioned in `SetupCompilerWarnings.cmake`, so that one still needs to be added there.

Reply:
Sorry, but that just doesn't seem to work either

Reply:
As seen below...